### PR TITLE
Settings page branch

### PR DESCRIPTION
**Pull request by Samuels-Account**
### Created a settings page that allows users to:

- Change their username
- Change their password
- Turn on/off notifications
- Make their Profile public
- Change Site theme(unsure?)

**The changes will be split into separate commits**

### CODE REVIEW

File: settingspage.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Fixed Sidebar Example</title>
```
Comment (JustaMushroom) [inline settingspage.html:5]: This title might need changing from "Fixed Sidebar Example"

File: .gitignore
Comment (JustaMushroom) [inline .gitignore:1]: Was deletion of this file intended? This could cause potential issues later on

File: README.md
Comment (JustaMushroom) [inline README.md:1]: Again, was deletion of this intended? It looks like it was removed in an unrelated commit

**Review by JustaMushroom — CHANGES_REQUESTED** [review]

File: settingspage.html (hunk 1 above)
Comment (Samuels-Account) [reply settingspage.html]: Fixed:
https://github.com/Samuels-Account/Recipe-Search/blob/90ebf919d04b8d687c0e81519750d287c8425a24/settingspage.html#L5
